src/components/tables/EntityTable.tsx

```diff
@@ -10,51 +12,208 @@ import {
+        null
+    );
+
+    const [unfilteredEntities, setUnfilteredEntities] = useState<
+        Entity[] | null
+    >(null);
+    const [filteredEntities, setFilteredEntities] = useState<Entity[] | null>(
+        null
+    );
+    const [entities, setEntities] = useState<Entity[] | null>(null);
+
     const [sortDirection, setSortDirection] = useState<SortDirection>('desc');
     const [columnToSort, setColumnToSort] =
-        useState<keyof EntityMetadata>('dateCreated');
+        useState<keyof Entity>('dateCreated');
 
     const [page, setPage] = useState(0);
     const rowsPerPage = 10;
     const rowHeight = 57;
 
-    const entityDetails: EntityMetadata[] = entities.map(
-        (entity) => entity.metadata
-    );
+    useEffect(() => {
+        (async () => {
+            const { data, error } = await supabaseClient
+                .from(TABLES.CONNECTORS)
+                .select(CONNECTORS_QUERY);
+
+            if (error) {
+                setTableState({
+                    status: TableStatuses.TECHNICAL_DIFFICULTIES,
+                    error,
+                });
+            } else {
+                setConnectors(data);
+            }
+        })().catch(() => {});
+
+        (async () => {
+            const { data, error } = await supabaseClient
+                .from(TABLES.LIVE_SPECS)
+                .select(LIVE_SPECS_QUERY)
+                .eq('spec_type', 'capture');
+
+            if (error) {
+                setTableState({
+                    status: TableStatuses.TECHNICAL_DIFFICULTIES,
+                    error,
+                });
+            } else {
+                setPublications(data);
+            }
+        })().catch(() => {});
+    }, [supabaseClient]);
+
+    useEffect(() => {
+        if (publications && connectors) {
+            if (publications.length > 0 && connectors.length > 0) {
+                const formattedPublication: Entity[] = publications.map(
+                    (publication) => {
+                        const catalogNamespace: string =
+                            publication.catalog_name;
+
+                        const dateCreated: string = publication.updated_at;
+
+                        const connectorInfo: ConnectorInfo | undefined =
+                            connectors.find(
+                                (connector) =>
+                                    connector.image_name ===
+                                    publication.connector_image_name
+                            );
+
+                        return {
+                            deploymentStatus: 'ACTIVE',
+                            name: catalogNamespace.substring(
+                                catalogNamespace.lastIndexOf('/') + 1,
+                                catalogNamespace.length
+                            ),
+                            catalogNamespace,
+                            connectorType: connectorInfo
```

Comment:
I was uncertain how we wanted to handle the scenario where only connector data failed to be retrieved. The easiest route was to populate the cell with a default value. `'Unknown'` was great for debugging, but an empty string is an option.